src/Forge/Forge.fsproj

```diff
@@ -60,6 +60,7 @@
     <Content Include="Tools/Paket/paket.bootstrapper.exe">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Compile Include="obj/Release/.NETFramework,Version=v4.5.AssemblyAttribute.fs" />
```

Comment:
hmm ... nasty ... that was included by the recommended `paket update` - my local build on linux had all tests pass

recomendations?